docs/containers/reference.md

```diff
@@ -13,6 +13,8 @@ The Docker extension includes several Visual Studio Code tasks to control the be
 
 The tasks allow for a great deal of control and customization. The final configuration is a combination of general defaults, platform-specific defaults (such as .NET Core and Node.js), and user input. User input takes precedence when it conflicts with defaults.
 
+All common features of Visual Studio Code tasks (for example, grouping tasks into compound tasks) are supported by Docker extension tasks. For more information on common task features and properties see [Visual Studio Code custom task documentation](https://code.visualstudio.com/docs/editor/tasks#_custom-tasks).
+
```

Comment:
This can be a relative link, which will allow navigation on GitHub and in VS Code.
Change to `(/docs/editor/tasks.md#custom-tasks)`
The VS Code website build preprocessor takes care of removing `.md` and adding the underscore for the published link.